src/main/java/com/android/volley/toolbox/CacheHeader.java

```diff
@@ -0,0 +1,148 @@
+package com.android.volley.toolbox;
+
+import androidx.annotation.Nullable;
+import androidx.annotation.VisibleForTesting;
+import com.android.volley.Cache;
+import com.android.volley.Header;
+import com.android.volley.VolleyLog;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Collections;
+import java.util.List;
+
+/** Handles holding onto the cache headers for an entry. */
+@VisibleForTesting
```

Comment:
this annotation doesn't make sense here; this class needs to be package-visible in order for other classes in this package to use it.

We use this annotation when we increase the visibility beyond what it otherwise would be for the _sole_ purpose of making it accessible in a unit test.

You should see Android Studio complaining about this, e.g. where DiskBasedAsyncCache is calling:

`cb.onGetComplete(entry.toCacheEntry(data));`

because it thinks toCacheEntry is only visible for testing, but here we're using it from prod code.

Reply:
fixed